crates/astria-composer/src/metrics_init.rs

```diff
@@ -0,0 +1,234 @@
+         lifetime"
+    );
+}
+
+pub const GETH_COLLECTOR_TRANSACTIONS_COLLECTED: &str = concat!(
+    env!("CARGO_CRATE_NAME"),
+    "_geth_collector_transactions_collected"
+);
+
+pub const GETH_COLLECTOR_TRANSACTIONS_DROPPED: &str = concat!(
+    env!("CARGO_CRATE_NAME"),
+    "_geth_collector_transactions_dropped"
+);
+
+pub const GETH_COLLECTOR_TRANSACTIONS_FORWARDED: &str = concat!(
+    env!("CARGO_CRATE_NAME"),
+    "_geth_collector_transactions_forwarded"
+);
+
+pub const GETH_COLLECTOR_CONNECTION_LATENCY: &str = concat!(
+    env!("CARGO_CRATE_NAME"),
+    "_geth_collector_connection_latency"
+);
+
+pub const GRPC_COLLECTOR_TRANSACTIONS_COLLECTED: &str = concat!(
+    env!("CARGO_CRATE_NAME"),
+    "_grpc_collector_transactions_collected"
+);
+
+pub const GRPC_COLLECTOR_TRANSACTIONS_DROPPED: &str = concat!(
+    env!("CARGO_CRATE_NAME"),
+    "_grpc_collector_transactions_dropped"
+);
+
+pub const GRPC_COLLECTOR_TRANSACTIONS_FORWARDED: &str = concat!(
+    env!("CARGO_CRATE_NAME"),
+    "_grpc_collector_transactions_forwarded"
+);
+
+pub const TRANSACTIONS_RECEIVED: &str = concat!(env!("CARGO_CRATE_NAME"), "_transactions_received");
+
+pub const TRANSACTIONS_DROPPED_TOO_LARGE: &str =
+    concat!(env!("CARGO_CRATE_NAME"), "_transactions_dropped_too_large");
+
+pub const NONCE_FETCH_COUNT: &str = concat!(env!("CARGO_CRATE_NAME"), "_nonce_fetch_count");
+
+pub const NONCE_FETCH_FAILURE_COUNT: &str =
+    concat!(env!("CARGO_CRATE_NAME"), "_nonce_fetch_failure_count");
+
+pub const NONCE_FETCH_LATENCY: &str = concat!(env!("CARGO_CRATE_NAME"), "_nonce_fetch_latency");
+
+pub const CURRENT_NONCE: &str = concat!(env!("CARGO_CRATE_NAME"), "_current_nonce");
+
+pub const TRANSACTION_SUBMISSION_LATENCY: &str =
+    concat!(env!("CARGO_CRATE_NAME"), "_transaction_submission_latency");
+
+pub const TRANSACTION_SUBMISSION_FAILURE_COUNT: &str = concat!(
+    env!("CARGO_CRATE_NAME"),
+    "_transaction_submission_failure_count"
+);
+
+pub const BUNDLES_SUBMISSION_SUCCESS_COUNT: &str = concat!(
+    env!("CARGO_CRATE_NAME"),
+    "_bundles_submission_success_count"
+);
+
+pub const BUNDLES_SUBMISSION_FAILURE_COUNT: &str = concat!(
+    env!("CARGO_CRATE_NAME"),
+    "_bundles_submission_failure_count"
+);
+
+pub const BUNDLES_OUTGOING_TRANSACTIONS_COUNT: &str = concat!(
+    env!("CARGO_CRATE_NAME"),
+    "_bundles_outgoing_transactions_count"
+);
+
+pub const BUNDLES_OUTGOING_BYTES: &str =
+    concat!(env!("CARGO_CRATE_NAME"), "_bundles_outgoing_bytes");
+
+pub const BUNDLES_NOT_DRAINED: &str = concat!(env!("CARGO_CRATE_NAME"), "_bundles_not_drained");
+
+pub const BUNDLES_DRAINED: &str = concat!(env!("CARGO_CRATE_NAME"), "_bundles_drained");
+
+pub const BUNDLES_TOTAL_COUNT: &str = concat!(env!("CARGO_CRATE_NAME"), "_bundles_total_count");
+
+pub const BUNDLES_TOTAL_BYTES: &str = concat!(env!("CARGO_CRATE_NAME"), "_bundles_total_bytes");
+
+pub const BUNDLES_TOTAL_TRANSACTIONS_COUNT: &str = concat!(
```

Comment:
Very highly correlated to the number of bundles submitted to the sequencer. Remove